meetings/2025-05-13-taskforce-meeting.md

```diff
@@ -0,0 +1,87 @@
+- Darius: I largely agree with Jaz. How does this work in ATProto with content labelers?
+- Jaz: You bring your own vocabulary, and the PDS can apply some number of actions... hide, blur, warn, maybe some other things. Labeler A can say "depiction of war" and "blur" and Labeler B can say "war images" and "hide". 
+- Emelia: https://docs.joinmastodon.org/entities/Filter/#filter_action mastodon now has the same three actions with content filtering 
+- Emelia: the difference between a label and content warning is the former is structured linked data that can be referenced, latter is unstrutured freeform text. For labels you have structured labels, but I'm not saying there should be one vocabulary but many vocabularies. Bluesky has a base set of labels that they have defined, Mastodon would probably also want to have theirs.
+- Darius: so a vocabulary should be an Unordered Collection of Label objects.
+- Emelia: yes, you can see it in FIRES: https://fires.fedimod.org/reference/protocol/labels.html
+- Emelia: I don't want us to be defining the labels. Just how you define the labels and share them, and how you associate a label with an object in the AP spec. You could cram all this into web annotations but I don't think that makes sense because we would be extending the web annotations vocabulary anyway since it doesn't have a notion of labels.
+- a: What if there are two labels, one says "war" and one says "depiction of war"? Who gets to decide two labels are the same thing? Maybe we have a two-tier system where a label publisher can suggest that their label is a subclass or equivalent label. 
+- Jaz: no one really finds equivalences in that ecosystem. It's more just a choice of who people subscribe to.
+- Emelia: right, we want to avoid string matching any more.
+- Jaz: why don't we just adopt what is on the FIRES link you posted?
+- Emelia: https://github.com/fedimod/fires/issues/68 issue on specifying recommended actions on content
+- Darius: do we need to specify actions as objects?
+- denny:  I don't see why specifying the action is within the ambit of a labeler's job. Wouldn't have occured to me as an application developer to look for action information in the label object. 
+- Emelia: I think you make it a string and we specify a set of starter strings
+- Darius: are we reinventing mime-types here?
+- Emelia: maybe we publish a repository of possible actions?
+- a: we probably don't want to take on too much at the start, so as a group here maybe we focus on representation and discovery and back off of the "what happens to the label". It seems like we have the basic building blocks for representation. For labels that clearly the object reference type system as agreed on by some Actor (Person/Service/etc), and then CW is more of freeform text and explicit warning rather than summary. Maybe we leave actions out of step 1
```

Comment:
```suggestion
- Emelia: Maybe we publish a repository of possible actions?
- a: We probably don't want to take on too much at the start, so as a group here maybe we focus on representation and discovery and back off of the "what happens to the label?" It seems like we have the basic building blocks for representation. For labels, that clearly reference the object type system as agreed on by some Actor (Person, Service, etc.), and then CW is more of freeform text and explicit warning rather than summary. Maybe we leave actions out of step 1.
```